### DISCUSSION TITLE
Optimize git commands

### DISCUSSION THREAD

**Discussion by JonnyDeates** [community/community#30568] [Other Feature Feedback, Questions, & Ideas]
As a zoomer, I have found using git push and git pull pretty irritating. I do not believe it is in the best interest as a developer to have to type two separate words to use core functionality of git. So I am proposing a solution:

- yoink -> git pull
- yeet -> git push

Now once you guys implement this, please mark that I am the reason you guys have changed this functionality, I would like royalties as well as a 3% equity in github.

**Comment by davidegiacometti** [comment]
If you are on PowerShell:
```
function yoink { git pull }
function yeet { git push }
```
Add functions to the profile to make them permanent. Refer to docs: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions?view=powershell-7.2

If you are on Linux I guess you can create an alias in the shell you are using.
